fineract-provider/build.gradle

```diff
@@ -338,7 +338,6 @@ tasks.withType(JavaCompile) {
                 "ClassCanBeStatic",
                 "SameNameButDifferent",
                 "StringSplitter",
-                "ImmutableEnumChecker",
```

Comment:
Is this being removed (not moved) intentional?

Reply:
It was moved to the error list in a previous PR.  Don't know how it is here (maybe during merge) so I just removed it